<launch>
  <node name="object_detector" pkg="object_detector" type="object_detector_node" output="screen">
    <remap from="image" to="/arm5e/image_rect_color" />
    <param name="use_training_image" type="bool" value="true"/>
    <param name="image_file_name" type="string" value="$(find object_detector)/testdata/trident_box.jpg"/>
    <param name="polygon_file_name" type="string" value="$(find object_detector)/testdata/trident_box_wide.poly"/>
    <param name="detection_threshold" value="0.5" />
    <param name="detector_config_file" value="$(find object_detector)/cfg/detector.cfg"/>
  </node>
  <node name="detection_display" pkg="object_detector" type="detection_display_node">
    <remap from="image" to="/arm5e/image_rect_color" />
    <remap from="detection" to="/object_detector/detection" />
  </node>
</launch>

